distribution/src/deb/bin/openhab.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
+
+# This script is meant to be used with the systemd startup.
+# The script relies on few environment variables to determine startup
+# behavior, those variables are:
+#
+# optional:
+#   OPENHAB_JAVA   --
+#   USER_AND_GROUP -- The user and group the openHAB is running as. The 
+#                     default is "openhab:openhab". In systemd environments
+#                     openhab.service must be adapted, when changing this.
+#   JAVA_ARGS      -- Additional arguments to the JVM.
+#   OPENHAB_ARGS   -- Additional arguments to the openHAB runtime.
+#   DEBUG          -- Start openHAB in debugging mode. The default is no, set 
+#                     it to yes for debugging purposes.
+#   HTTP_PORT      -- 
+#   HTTPS_PORT     -- 
+#   TELNET_PORT    -- 
+# Normaly systemd provides this variable using the /etc/default/openhab file.
+
+include="`dirname "$0"`"/openhab.in.sh
+if [ -r "$include" ]; then
+	. "$include"
+else
+	echo "Error $include not found or not readable" >&2
+	exit 1
+fi
+
+if [ -z $HTTP_PORT ]; then
+	HTTP_PORT=8080
+fi
+if [ -z $HTTPS_PORT ]; then
+	HTTPS_PORT=8443
+fi
+if [ -z $TELNET_PORT ]; then
+	TELNET_PORT=5555
+fi
+
+JAVA="/usr/bin/java"
+SETOWNER="no"
+
+LAUNCHER=`ls ${OPENHAB_DIR}/server/plugins/org.eclipse.equinox.launcher_*.jar`
```

Comment:
As long as there is only one org.eclipse.equinox.launcher jar in ${OPENHAB_DIR} it will work.
The "find  | sort | tail" from start.sh will work even if there are several launcher jars in ${OPENHAB_DIR} and it will traverse directories and find also launchers located in sub dirs. I think this is not needed for starting openHAB, is it?